File: Auto-generated-tasks/Difficulty-3/Task-4/Solution.pas
Var
	NN := 6, t, Qwerty:=0, Rez:=0, help1, help2: integer;
	S: Array[0..NN] of integer;
Begin
	randomize();
	for t:= 0 to NN do begin
		S[t] := random(5);
		WriteLn("S[", t, "] = ", S[t]);
		Qwerty := Qwerty + S[t];
		if (S[t] > Rez) then
			Rez := S[t];
	end;
	Qwerty := Qwerty / NN;
	for t:= 0 to NN do begin
		help1 := S[t] - Qwerty;
		help2 := Qwerty - S[t];
		if ((help1 >= 0) and (help1 < Rez - Qwerty)) then
			Rez := S[t];
		else if ((help2 >= 0) and (help2 > Rez))
	        Rez := S[t];
	end;
	WriteLn("Rez = ", Rez);
End.

File: Auto-generated-tasks/Difficulty-3/Task-4/Solution.py
from random import *
NN = 6
t = 0
Qwerty = 0
Rez = 0
S = [0] * NN
for t in range(0, NN):
	S[t] = randint(0, 5)
	print("S[", t, "] = ", S[t])
	Qwerty += S[t]
	if (S[t] > Rez):
		Rez = S[t]
Qwerty /= NN
for t in range(0, NN):
	help1 = S[t] - Qwerty
	help2 = Qwerty - S[t]
	if ((help1 >= 0) && (help1 < Rez - Qwerty)):
		Rez = S[t]
	else if ((help2 >= 0) && (help2 > Rez)):
        Rez = S[t]
print("Rez = ", Rez)